### DISCUSSION TITLE
Can you make an online version such as a Chrome extension or a local deployment version that doesn't rely on SD

### DISCUSSION THREAD

**Discussion by LunarHeron** [DominikDoom/a1111-sd-webui-tagcomplete#264] [Ideas]
Now I use novelai a lot, but because tagcomplete is so good, I have to open SD, which is a waste of system resources, I would like to ask the author if he can make an online version such as Chrome extension or local deployment version that does not rely on SD

**Comment by DominikDoom** [comment]
This extension is pretty reliant on the webui in its base structure, so a standalone version would be a bigger rework and something I don't really have the time to work on in the near future. Maybe later this year, but I can't promise it, sorry.
One workaround, while not perfect, would be to activate the option to start the webui without loading a model, which is possible in the latest version. That would at least reduce the resources it requires.

**Reply by LunarHeron** [reply]
Thanks for your reply, I temporarily put the csv file in the Google office, which is slightly less efficient
